base/src/test/proto/spine/test/validate/msg/messages.proto

```diff
@@ -242,14 +242,25 @@ message MessageWithMapDoubleField {
 
 // Messages for testing validation rules.
 
-message ValidationRule {
-    option (validation_of) = "spine.test.validate.msg.ValidationRuleTarget.can_be_valid";
+message AValidationRule {
+    option (validation_of) = "spine.test.validate.msg.FirstRuleTarget.can_be_valid,"
+                             "spine.test.validate.msg.SecondRuleTarget.can_be_valid";
```

Comment:
What if there is a space before `spine.test.validate.msg.SecondRuleTarget.can_be_valid`? E.g.:
```
"    spine.test.validate.msg.SecondRuleTarget.can_be_valid"
```
The feature should work fine (you should've stripped the whitespace). However I don't see a test covering this case.